appleid.md

```diff
@@ -0,0 +1,27 @@
+# Apple ID
+Apple IDs are a requirement for Macbook support. 
+## Installation
+You will receive an invitation via email containing 
+* Your managed Apple ID eg: user@studio24.net.
+* Your temporary password
+
+
+* Click the link in the email or visit the URL [Apple Business Manager](https://business.apple.com)
```

Comment:
May be worth having this not in a bullet and in a paragraph

Reply:
Updated